README.md

```diff
@@ -26,6 +26,20 @@ and then enable it in your `config.toml`:
 theme = "hyde"
 ```
 
+## Folder tree
+This theme expects all the articles to be in the root of the folder. The expected `content` folder tree is as follow
+```
+├───content
+│   │   _index.md
+│   ├───content_1
+│   │       index.md
+│   ├───content_2
+│   │       index.md
```

Comment:
you don't need to nest those if you don't have colocated assets. You can  have `_index.md`, `content_1.md` and `content_2.md` in the content folder directly

Reply:
Yep if you want assets like images yes. It's just that in this example, there aren't any so it looks like you have to create a folder

Reply:
I see, I've just amended the commit, added image under folder.